Patch by Cyril Brulebois <cyril.brulebois@enst-bretagne.fr>
fixes an FTBFS on GNU/kFreeBSD
see bug 415106.


--- par2cmdline-0.4/diskfile.cpp	2007-03-16 06:56:17.225694000 +0100
+++ par2cmdline-0.4/diskfile.cpp	2007-03-16 06:58:10.000000000 +0100
@@ -27,6 +27,11 @@
 #endif
 #endif
 
+#if defined(__FreeBSD_kernel__)
+#include <sys/disk.h>
+#define BLKGETSIZE64 DIOCGMEDIASIZE
+#endif
+
 
 #ifdef WIN32
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////
